website_snippet_data_slider/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Website Snippet - Data Slider",
+    "summary":
+        "Abstract data slider for use on website."
+        "  Primary use (and default implementation) is product slider.",
+    "version": "11.0.1.0.0",
+    "category": "Website",
+    "website": "https://laslabs.com/",
+    "author": "LasLabs, Odoo Community Association (OCA), Kaushal Prajapati",
```

Comment:
OCA should stay at the end, always. Plus, we don't put 'migrators' inside authors AFAIK